lib/spack/spack/build_systems/intel.py

```diff
@@ -188,5 +982,29 @@ def save_silent_cfg(self):
         """Copies the silent.cfg configuration file to ``<prefix>/.spack``."""
         install('silent.cfg', join_path(self.prefix, '.spack'))
 
+    @run_after('install')
+    def uninstall_ism(self):
+        # The "Intel(R) Software Improvement Program" [ahem] gets installed,
+        # apparently regardless of PHONEHOME_SEND_USAGE_DATA.
```

Comment:
Is this the cause of that mysterious `intel` directory that keeps popping up in my home directory? This is a very nice addition, I would love to see this thing disappear.

Reply:
Yes, I believe it is.

I put this in on (a) general principles, (b) I would want neither my users nor the cluster to be burdened with the ism monitoring, and (c) I have seen runaway ism daemon loads. My compute nodes aren't routed out, so there's no point for ism to even be there.